esp/src/src/nls/hpcc.ts

```diff
@@ -1130,7 +1130,7 @@ export = {
         WSDL: "WSDL",
         WUID: "WUID",
         Wuidcannotbeempty: "Wuid Cannot Be Empty.",
-        WUSnapshot: "WU Snapshot",
+        WuSnapShot: "WU SnapShot",
```

Comment:
I'd change the casing on `WUSnapShot: "WU SnapShot"` to be `WUSnapShot: "WU Snapshot"`. Looks good otherwise